src/Highlighter.ts

```diff
@@ -215,11 +218,16 @@ export class CodeHighlighter {
 		container.innerHTML = toHtml(this.themeMapper.fixAST(result.renderedGroupAst));
 	}
 
-	getHighlightTokens(code: string, lang: string): TokensResult | undefined {
+	async getHighlightTokens(code: string, lang: string): Promise<TokensResult | undefined> {
 		if (!this.obsidianSafeLanguageNames().includes(lang)) {
 			return undefined;
 		}
-
+		// load bundled language ​​when needed
+		try {
```

Comment:
I dislike this try catch. Is there a way to check if a language is loaded? Otherwise, maybe track the loaded languages in some record.